fastlane/Fastfile

```diff
@@ -450,58 +508,64 @@ UPLOAD_TO_PLAY_STORE_JSON_KEY = File.join(PROJECT_ROOT_FOLDER, '.configure-files
+    comment_body = prototype_build_details_comment(
+      app_display_name: 'Simplenote Android',
+      download_url: install_url,
+      metadata: {
+        'Build Type': prototype_build_type
+      }
     )
 
+    unless ENV['BUILDKITE_PULL_REQUEST'].nil?
+      comment_on_pr(
+        project: GITHUB_REPO,
+        pr_number: Integer(ENV.fetch('BUILDKITE_PULL_REQUEST', nil)),
+        reuse_identifier: 'simplenote-prototype-build-link',
+        body: comment_body
+      )
+    end
+
     if ENV['BUILDKITE']
-      message = "Simplenote Installable Build: [#{filename}](#{install_url})"
-      sh('buildkite-agent', 'annotate', message, '--style', 'info', '--context', "installable-build")
+      message = "Simplenote Android Prototype Build: [#{filename}](#{install_url})"
+      sh('buildkite-agent', 'annotate', message, '--style', 'info', '--context', 'prototype-build')
     end
   end
 
-
   #####################################################################################
   # Screenshots lanes
   #####################################################################################
 
-  desc "Take screenshots in the app"
-  lane :take_screenshots do |options|
-    package = "com.automattic.simplenote.debug"
+  desc 'Take screenshots in the app'
+  lane :take_screenshots do |_options|
+    package = 'com.automattic.simplenote.debug'
 
     # Reset the app in the emulator, just to avoid possible state related
     # issues
     begin
       adb(command: "shell pm clear #{package}")
-    rescue
-      UI.message("Failed to reset app data on emulator. This can happen when the app is not installed yet.")
+    rescue StandardError
+      UI.message('Failed to reset app data on emulator. This can happen when the app is not installed yet.')
     end
 
     # We use the screenshots build type for the app APK, which allows us to not
     # set the special permissions in the debug build type so that it doesn't
     # deviate from the release one. For the test APK though, we still use the
     # debug build type, as it's the only testable one, and there's no need to
     # make a dedicated one for the screenshots.
-    gradle(task: "assemble", build_type: "Screenshots")
-    gradle(task: "assemble", build_type: "DebugAndroidTest")
+    gradle(task: 'assemble', build_type: 'Screenshots')
+    gradle(task: 'assemble', build_type: 'DebugAndroidTest')
 
     capture_android_screenshots(
-      locales: ["en-US"],
+      locales: ['en-US'],
       output_directory: screenshots_directory,
       clear_previous_screenshots: false,
       app_package_name: package,
-      app_apk_path: "Simplenote/build/outputs/apk/screenshots/Simplenote-screenshots.apk",
-      tests_apk_path: "Simplenote/build/outputs/apk/androidTest/debug/Simplenote-debug-androidTest.apk",
+      app_apk_path: 'Simplenote/build/outputs/apk/screenshots/Simplenote-screenshots.apk',
+      tests_apk_path: 'Simplenote/build/outputs/apk/androidTest/debug/Simplenote-debug-androidTest.apk',
```

Comment:
Use `File.join` instead

Reply:
Fixed in https://github.com/Automattic/simplenote-android/commit/3267c3d07bb4efb667be614d849bb73e9b2b9ac3